warehouse/query-core/src/main/java/datawave/query/iterator/SeekableIterator.java

```diff
@@ -9,6 +9,7 @@
 /**
  *
  */
```

Comment:
Small note. Would you consider adding a little note in the javadoc to link a reference to the replacing class? like:

"use {@link #new()} instead." - something like that

Reply:
Is this interface still being used somewhere?

Reply:
I don't think this interface is still used anywhere, but just in case it is I've deprecated it and marked it for removal so that any project that does use it will see build warnings.